subsys/bluetooth/audio/csip_set_coordinator.c

```diff
@@ -300,6 +300,17 @@ static void lock_changed(struct bt_csip_set_coordinator_csis_inst *inst, bool lo
 	}
 }
 
+static void sirk_changed(struct bt_csip_set_coordinator_csis_inst *inst)
+{
+	struct bt_csip_set_coordinator_cb *listener;
+
+	SYS_SLIST_FOR_EACH_CONTAINER(&csip_set_coordinator_cbs, listener, _node) {
+		if (listener->sirk_changed) {
+			listener->sirk_changed(inst);
```

Comment:
It cannot :) And even if it can, then it would provide `NULL` to the callback (which isn't documented, but that's because it isn't intentionally possible).